apparmor.d/profiles-m-r/mutt

```diff
@@ -0,0 +1,178 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2023 Zane Zakraisek <zz@eng.utah.edu>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}bin/mutt
+profile mutt @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/nameservice>
+  include <abstractions/openssl>
+  include <abstractions/consoles>
+
+  @{exec_path} mr,
+
+  # Mutt config files
+  /usr/{local/,}share/mutt/Muttrc r,
+  /etc/{mutt/,}Muttrc             r,
+  owner @{HOME}/.mutt/**          r,
```

Comment:
Simplify as owner `@{HOME}/.mutt/{,**} rw,`, and remove the later rules (`owner @{HOME}/.mutt*/`).

Reply:
I don't think mutt should necessarily have write access to everything underneath `@{HOME}/.mutt/`
Ah but I see the mistake I'll tighten up the later rules.

Reply:
`owner @{HOME}/.mutt/**` is for mutt's misc config files, so this should be read only.
`owner @{HOME}/.mutt*/{,*}` is used for temporary files created by mutt.